docs/source/blackbox/score_decrease.rst

```diff
@@ -0,0 +1,91 @@
+.. _eli5-decrease-in-score:
+
+Decrease in Score
+=================
+
+eli5 provides a way to compute feature importances for any black-box
+estimator by measuring how score decreases when a feature is not available.
+
+A similar method is described in Breiman, "Random Forests", Machine Learning,
+45(1), 5-32, 2001 (available online at
+https://www.stat.berkeley.edu/%7Ebreiman/randomforest2001.pdf) as
+"mean decrease in accuracy".
+
+The idea is the following: feature importance can be measured by looking at
+how much the score (accuracy, F1, R^2, etc. - any score we're interested in)
+decreases when a feature is not available. To do that one can remove feature
+from the dataset, re-train the estimator and check the score. But it requires
+re-training an estimator for each feature, which can be computationally
+intensive.
+
+To avoid re-training the estimator we can remove a feature only from the
+test part of the dataset, and compute score without using this
+feature. It doesn't work as-is, because estimators expect feature to be
+present. So instead of removing a feature we can replace it with random
+noise - feature column is still there, but it no longer contains useful
+information. This method works if noise is drawn from the same
+distribution as original feature values (as otherwise estimator may
+fail). The simplest way to get such noise is to shuffle values
+for a feature, i.e. use other example's feature value - this is how
+"decrease in score" feature importances computed.
+
+For sklearn-compatible estimators eli5 provides
+:class:`~.ScoreDecreaseFeatureImportances` wrapper; if you want to use this
+method for other estimators there is :mod:`eli5.score_decrease` module.
+
+For example, this is how you can check feature importances of
+`sklearn.svm.SVC`_ classifier, which is not supported by eli5 directly
+when a non-linear kernel is used::
+
+    import eli5
+    from eli5.sklearn import ScoreDecreaseFeatureImportances
```

Comment:
Oh sorry, my bad - I thought there is such a module and this was a typo, but now I see that it's only a folder in docs.